kratos/geometries/geometry.h

```diff
@@ -647,11 +647,46 @@ class Geometry
     ///@name Operations
     ///@{
 
-    virtual Pointer Create( PointsArrayType const& ThisPoints ) const
+    /**
+     * @brief Creates a new geometry pointer
+     * @param ThisPoints the nodes of the new geometry
+     * @return Pointer to the new geometry
+     */
+    virtual Pointer Create(
+        PointsArrayType const& ThisPoints
+    ) const
     {
         return Pointer( new Geometry( ThisPoints, mpGeometryData ) );
     }
 
+    /**
+     * @brief Creates a new geometry pointer
+     * @param NewGeometryId the ID of the new geometry
+     * @param ThisPoints the nodes of the new geometry
+     * @return Pointer to the new geometry
+     */
+    virtual Pointer Create(
+        IndexType NewGeometryId,
+        PointsArrayType const& ThisPoints
```

Comment:
Can we use here Geometry instead of the PointsArrayType? 

That would allow to use the new geometries which have additional data enhenced.

Reply:
Sure

Reply:
@tteschemacher done